website/institutions/model.py

```diff
@@ -128,7 +129,7 @@ def find_one(cls, query=None, deleted=False, **kwargs):
     def load(cls, key):
         from website.models import Node
         try:
-            node = Node.find_one(Q('institution_id', 'eq', key), allow_institution=True)
+            node = Node.find_one(Q('institution_id', 'eq', key) & Q('is_deleted', 'ne', True), allow_institution=True)
```

Comment:
I think I agree. We typically don't try to hide `is_deleted` at the model level. Querying at the view layer will keep consistency with other models.